Topics/Software_Engineering/Software_Licenses.md

```diff
@@ -0,0 +1,134 @@
+
+### Propietary licenses
+
+We have mentioned multiple times these types of licenses. To give a proper definition, prorpriatry licenses, also known as commercial licenses, are the most restrictive types of licenses. It is the opposite of free and open source licenses. Software that falls under these licenses cannot be modified or distributed to the public community. The source code is considered a trade secret. Prorpietary licenses provide the most legal protection to owners against unauthorised use of software. They fully protects the owners' inteelectual property rights. To use a software that falls under this category of licenses, users have to sign a End-User License Agreement (EULA). For instance, Microsoft Windows is a software with prorpieatry license. The latters restrics users from actions like from reverse engineering, and distributing the software to multiple users. if you do not accept the EULA, then Microsoft prevents you from using its products. 
+
+
+Here is a picture summarizing the various available types of software licenses:
+
+![Licenses](https://cdn.ttgtmedia.com/rms/onlineimages/5_types_of_software_licenses-f.png)
+
+## How to choose the right license?
+
+With so many possible options, it can be daunting to choose the license that is approapiate to your software, project, and team. 
+
+Firstly, you should choose between the two big categories; whether you want your license to belong to the free and open source licenses or to the proprietary licenses. If you are not planning on commercializing your application, then it would be a better idea to opt for an open source license.
+
+Among the open source licenses, if you are comfortable with someone else taking you code, changing it and making it propretsry (close), then you should go for the popular option of permissive licenses. On the other hand, if you do not want this, then using a copyleft license might be the better choice. 
+
+For more information, check this [website](https://choosealicense.com/) created by GitHub that further explains options available to you.
+
+
+## Breach of Licenses
+
```

Comment:
Consider adding a brief introduction explaining what this section is about as the transition from other content is otherwise a bit abrupt.